docs/api/fieldarray.md

```diff
@@ -67,7 +67,7 @@ export const FriendList = () => (
 
 ### `name: string`
 
-The name or path to the relevant key in [`values`](./formik.md#values--field-string-any-).
+The name or path to the relevant key in [`values`](./formik#values--field-string-any-).
```

Comment:
I didn't remote .md extension because, in the website src code, the extension will be remote before transforming to a link
and keep .md extension will help navigate around directly in the md file correctly.